src/bin/index.ts

```diff
@@ -147,14 +148,22 @@ async function run(args: CliArgs) {
   // lint package
   await lint(cwd)
 
+  const spinner = ora({
+    text: 'bundling...\n\n',
+    spinner: 'dots',
+    color: 'green',
+  })
```

Comment:
LMK how you'd like the output to behave.
Current:

https://github.com/huozhi/bunchee/assets/120007119/78c36765-99fe-452a-98f1-0e164b2d9e86